src/mmw/js/src/draw/views.js

```diff
@@ -96,6 +96,10 @@ var SelectAreaView = Marionette.ItemView.extend({
         ofg.on('layerremove', _.bind(this.clearLabel, this));
     },
 
+    onRender: function() {
+        $(this.el).find('i.split').popover({ trigger: 'hover' });
```

Comment:
Consider adding an item to the `ui` attribute for the help icon for each of the modified views.